src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions.mdx

```diff
@@ -665,7 +665,7 @@ As noted in our [basic NRQL syntax doc](/docs/query-your-data/nrql-new-relic-que
 
     Use the `LIMIT` clause to control the maximum number of facet values returned by `FACET` queries or the maximum number of items returned by `SELECT *` queries. This clause takes a single integer value as an argument. If the `LIMIT` clause is not specified, or no value is provided, the limit defaults to 10 for `FACET` queries and 100 in the case of `SELECT *` queries.
 
-    The maximum allowed value for the `LIMIT` clause is 2,000.
+    The maximum allowed value for the `LIMIT` clause is 5,000. Queries can use the `LIMIT MAX` clause instead of a specific value to automatically use the current maximum value. The maximum value may change in the future. If you want your query's behavior to remain unchanged, specify an explicit value instead of using `LIMIT MAX`.
```

Comment:
```suggestion
    The maximum allowed value for the `LIMIT` clause is 5000. Queries can use the `LIMIT MAX` clause instead of a specific value, which automatically defaults to the current maximum value. You can use this if you always want to post the maximum number of results, even if it changes in the future. If you want your query's behavior to remain unchanged, specify an explicit value instead of using `LIMIT MAX`.
```